fluent-bit/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
https://helm.sh/docs/topics/charts/
>The apiVersion field should be v2 for Helm charts that require at least Helm 3

If `apiVersion` is v2,  dose makefile work ? 
https://github.com/chatwork/charts/pull/146/files#diff-8787ed5594febb4855b1641633af1e8cR6

Reply:
Looks like CI is through😁
https://circleci.com/gh/chatwork/charts/1150?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link